spec/import_result_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe ActiveAdminImport::ImportResult do
+  context "failed_message" do
+    let(:import_result) { ActiveAdminImport::ImportResult.new }
+
+    before do
+      require'pry';binding.pry
+      Author.create(name: 'John', last_name: 'Doe')
+      Author.create(name: 'Jane', last_name: 'Roe')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.